sdk/js/examples/example_for_browser.html

```diff
@@ -8,29 +8,51 @@
 <h3>Use library directly with &lt;script&gt; tag</h3>
 <script src="../sentience.min.js"></script>
 <script>
-
-    const EXAMPLE_VALID = {
-        hash: '5cff951d70ad5d4da7c12187331d98eabbf4023f7aeb547e949224ddd1420fc7',
-        public_key: '835cc0e84c2a5190561d7c2eaf10eb2597cbe7a71541084c5edea32b60bc5e68',
-        signature: '819218f6ad5a8ec86091afc1dfbecc5d21384d62f1df0c649ab0c351e515e825fda72d124091418b3cc4a3be35b20dd471b8f8447a448712315b19ecaef34206',
-    }
-
-    const EXAMPLE_INVALID_HASH = {
-        hash: '5cff951d70ad5d4da7c12187331d98eabbf4023f7aeb547e949224ddd1420fc8',
-        public_key: '835cc0e84c2a5190561d7c2eaf10eb2597cbe7a71541084c5edea32b60bc5e68',
-        signature: '819218f6ad5a8ec86091afc1dfbecc5d21384d62f1df0c649ab0c351e515e825fda72d124091418b3cc4a3be35b20dd471b8f8447a448712315b19ecaef34206',
-    }
-
     //replace console.log with document.write to make the behavior visible
     console.log = (m) => {
         document.write(`${m}<br>`);
     };
 
-    console.log("Sentience.verifySignature(EXAMPLE_VALID):")
-    console.log(Sentience.verifySignature(EXAMPLE_VALID));
+    // TODO:
+    const GALADRIEL_API_KEY = "TODO"
```

Comment:
Did not figure out how to add env variables to html

Reply:
I mean you cant, its client-side code, client needs to input it somehow

Reply:
could use just <input> and JS to input it I guess